### DISCUSSION TITLE
Process for adding a new library to this organization

### DISCUSSION THREAD

**Discussion by axelson** [beam-community/beam-community.org#23] [General]
Hi! Is there a process to add a new library to this organization?

I looked at https://github.com/beam-community and https://beam-community.org/ but I didn't see anything.

I'd like to recommend moving public_suffix under this umbrella. The library hasn't seen an update since 2018 and it only needs a small amount of maintenance. I started a discussion about transferring ownership at: https://github.com/seomoz/publicsuffix-elixir/issues/37 

If the library is moved under beam-community I'd be interested in helping maintain it, and @reisub has expressed interest as well: https://github.com/axelson/publicsuffix-elixir/issues/2

What do you all think?

**Comment by doomspork** [comment]
Hey @axelson this is a _great_ thread and we probably should formalize a document for public consumption. To date the process has been pretty ad-hoc. The biggest concern I have with taking on new projects is going to be maintainers. There's 27 people in the org but I'd say less than 5 are currently active in these repos and I don't want to put _more_ on the plates of a few.

If yourself and @reisub are open to helping maintain and the current maintainer is willing to transfer the repo _and_ the Hex package permissions, I'm open to supporting this.

**Reply by gilbertwong96** [reply]
Hi, @doomspork, is there any update on this thread ?

**Comment by gilbertwong96** [comment]
In this issue https://github.com/xou/elixlsx/issues/163#issuecomment-2974362120, we discussed that transfer the https://github.com/xou/elixlsx into this organization: https://github.com/beam-community. How do we make it?

**Reply by doomspork** [reply]
Howdy @gilbertwong96! I see @snewcomer and you already spoke. Since he's willing to help maintain we'd be happy to have the project transferred but we'd also need to added to hex.pm as an owner.

You can add myself (doomspork on hex and github) to the package or I believe you can add the org https://hex.pm/orgs/beam_community.

If you let me know once you've added us on hex and initiated the repo transfer on github, we can get things sorted!
